setup.py

```diff
@@ -28,5 +28,5 @@
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
     ],
-    install_requires=['six', 'lxml', 'scikit-learn', 'tldextract', 'requests'],
+    install_requires=['six', 'lxml', 'scikit-learn', 'tldextract', 'requests', 'GitPython'],
```

Comment:
There is no need for `GitPython` anymore

Reply:
Thanks!